test/saithrift/tests/switch.py

```diff
@@ -103,7 +103,7 @@ def sai_thrift_create_fdb(client, vlan_id, mac, port, mac_action):
     fdb_attribute3_value = sai_thrift_attribute_value_t(s32=mac_action)
     fdb_attribute3 = sai_thrift_attribute_t(id=SAI_FDB_ENTRY_ATTR_PACKET_ACTION,
                                             value=fdb_attribute3_value)
-    fdb_attr_list = [fdb_attribute1, fdb_attribute2, fdb_attribute3]
+    fdb_attr_list = [fdb_attribute1, fdb_attribute2]#, fdb_attribute3]
```

Comment:
When PACKET ACTION is set to FORWARD it is creating l2 entry with DiscardDest with which we faced some issues when we tried use cases of routing between vlans.

Will add the same now
Thanks